datafusion/Cargo.toml

```diff
@@ -69,6 +70,8 @@ regex = { version = "^1.4.3", optional = true }
 lazy_static = { version = "^1.4.0", optional = true }
 smallvec = { version = "1.6", features = ["union"] }
 rand = "0.8"
+avro-rs = { version = "0.13", features = ["snappy"], optional = true }
+num-traits = "0.2"
```

Comment:
I am not seeing `num_traits` being used in the code?

Reply:
It looks like num-traits should be marked as optional and be included as part of `avro` feature flag?

Reply:
```suggestion
num-traits = { version = "0.2", optional = true }
```